recipes/libjpeg-turbo/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "1.5.1" %}
+{% set libjpeg_version = "8c" %}
+
+package:
+    name: libjpeg-turbo
+    version: {{ libjpeg_version }}_{{ version }}
```

Comment:
So is @valgur trying to version this package according to the `libjpeg` version used to compile it?

If this is the case, I disagree with this practice because it'd give place to the same problem we are discussing in https://github.com/conda-forge/boost-feedstock/pull/15, i.e. to mix conda-forge and defaults compiled packages.